cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/dependabot.yml

```diff
@@ -1,12 +1,45 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
-  target-branch: development
-  labels:
-  - DEPENDABOT
-  commit-message:
-    prefix: "[DEPENDABOT]"
+    - package-ecosystem: github-actions
+      directory: "/"
+      schedule:
+          interval: daily
+      open-pull-requests-limit: 10
+      target-branch: development
+      labels:
+          - DEPENDABOT
+      commit-message:
+          prefix: "[DEPENDABOT]"
+
+    - package-ecosystem: pip
```

Comment:
out of curiosity, why is this needed, like pip in GA?

Reply:
pip=poetry in dependabots terms. It just needs to know what to look for

Reply:
I see